bin/smoke-test-rest-proxy.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+retries=5
+
+for ((i = 0; i <= retries; i++)); do
+    response=$(curl --silent --verbose --fail http://localhost:8083/topics)
+
+    if [[ $response == '["_schemas","__consumer_offsets"]' ]]; then
```

Comment:
Would `__consumer_offsets` be enough here, the `_schemas` is schema registry special.

Reply:
Also this is existing code/logic.